src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -84,6 +85,27 @@
         return emails;
     }
     
+    /**
+     * Generates the feedback submission confirmation email for the given {@code session} for {@code student}
+     * and {@code instructor}.
+     */
+    public EmailWrapper generateFeedbackSubmissionConfirmationEmails(
+            FeedbackSessionAttributes session, StudentAttributes student,
+            InstructorAttributes instructor) {
+        CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
+        EmailWrapper email = new EmailWrapper();
+        
+        if (instructor != null) {
+            email = generateFeedbackSessionEmailBaseForInstructorSubmissionConfirmation(course, session,
+                    instructor);
+        }
+        if (student != null) {
+            email = generateFeedbackSessionEmailBaseForStudentSubmissionConfirmation(course, session,
+                    student);
+        }
+        return email;
```

Comment:
To view `studentFeedbackSubmissionEditPage` user must be logged in or can be unregistered student with regKey ?
I will add Assert statement to check that both userId and unregisteredStudent emails are not empty ? This will assure that Instructor and students are both not null ?